src/rust/aggregator/src/pipelines/update_materialized_view.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::anyhow;
+use chrono::{DateTime, Duration, Utc};
+use sqlx::PgPool;
+
+use aggregator::{Pipeline, PipelineAggregationResult, PipelineError};
+
+pub struct UpdateMaterializedView {
+    pool: PgPool,
+    view_name: String,
+    update_interval: std::time::Duration,
+    last_indexed_timestamp: Option<DateTime<Utc>>,
+}
+
+#[allow(dead_code)]
+impl UpdateMaterializedView {
+    pub fn new(
+        pool: PgPool,
+        view_name: impl Into<String>,
+        update_interval: std::time::Duration,
+    ) -> Self {
+        Self {
+            pool,
+            view_name: view_name.into(),
+            update_interval,
+            last_indexed_timestamp: None,
+        }
+    }
+}
+
+#[async_trait::async_trait]
+impl Pipeline for UpdateMaterializedView {
+    fn model_name(&self) -> String {
+        String::from("UpdateMaterializedView")
```

Comment:
`RefreshMaterializedView` ?

Reply:
:+1:

Reply:
Note this still says `Update...` as of 42ed45f3

I highlighted this one separately before as failing to change it wouldn't produce a compiler error